src/main/java/sirius/biz/storage/layer2/BasicBlobStorageSpace.java

```diff
@@ -1408,12 +1407,14 @@ private V awaitConversionResultAndRetryToFindVariant(B blob, String variantName,
      * @param variant the variant to generate
      */
     private void invokeConversionPipelineAsync(B blob, V variant) {
-        Watch watch = Watch.start();
-        conversionEngine.performConversion(blob, variant.getVariantName()).onSuccess(handle -> {
-            try (FileHandle automaticHandle = handle) {
+        conversionEngine.performConversion(blob, variant.getVariantName()).onSuccess(handleAndWatch -> {
+            try (FileHandle automaticHandle = handleAndWatch.getFirst()) {
                 String physicalKey = keyGenerator.generateId();
                 getPhysicalSpace().upload(physicalKey, automaticHandle.getFile());
-                markConversionSuccess(variant, physicalKey, automaticHandle.getFile().length(), watch.elapsedMillis());
+                markConversionSuccess(variant,
+                                      physicalKey,
+                                      automaticHandle.getFile().length(),
+                                      handleAndWatch.getSecond().elapsedMillis());
```

Comment:
Is this enough added precision for now or do we want to track download, conversion and upload duration separately right away?

Reply:
Well we could add a 2nd measurement for the "overall time" - this would als include the queuing time...